src/middlewared/middlewared/plugins/alert.py

```diff
@@ -328,17 +328,17 @@ async def list_sources(self):
         Str('uuid'),
         Str('source'),
         Str('klass'),
-        Any('args'),
+        Any('args', null=True),
         Str('node'),
-        Str('key'),
+        Text('key', null=True),
         Datetime('datetime'),
         Datetime('last_occurrence'),
         Bool('dismissed'),
         Any('mail', null=True),
         Str('text'),
         Str('id'),
         Str('level'),
-        Str('formatted', null=True),
+        Text('formatted', null=True),
```

Comment:
@sonicaj I suppose we now have middleware failing to return strings with length > 1024.